Content.Shared/Borgs/LawsComponent.cs

```diff
@@ -0,0 +1,34 @@
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Borgs
+{
+    [RegisterComponent, NetworkedComponent]
+    public sealed class LawsComponent : Component
+    {
+        [DataField("laws")]
+        public List<string> Laws = new List<string>();
```

Comment:
This fails to handle:
- Ion laws
- Negative law indexes
- Law 0
I recommend using a SortedDictionary<int, (string, LawProperties)> where LawProperties is an enum for things like being an ion law. Additionally this entire class should only be writable to SharedLawsSystem.

Reply:
should I write a type serializer for this then